STYLE.md

```diff
@@ -0,0 +1,2 @@
+* line lengths <= 100 characters
```

Comment:
Yep, this works. Just so we have something to reference. I would also leave a note in the README with the link to the style guide.

Reply:
Next time I update either, I'll add it.